framework/quota/src/org/apache/cloudstack/quota/vo/ServiceOfferingVO.java

```diff
@@ -312,7 +271,7 @@ public String getDeploymentPlanner() {
     public String getDetail(String name) {
         assert (details != null) : "Did you forget to load the details?";
```

Comment:
I would have solved this one by removing the assert. Do we even run any tests with asserts enabled? (no :-1: intended!)